staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go

```diff
@@ -335,18 +351,37 @@ func newAzureTokenSourceDeviceCode(environment azure.Environment, clientID strin
 	if apiserverID == "" {
 		return nil, errors.New("apiserver-id is empty")
 	}
+	if omitTokenSPNPrefixString != "" {
```

Comment:
I think we can do this conversion [here](https://github.com/kubernetes/kubernetes/pull/87630/files#diff-0dc9d8034d41e3eb9e18333d16b90035R82) and pass `omitTokenSPNPrefix bool` as a parameter to this function. We are doing similar approach for the azure environment.

Reply:
okay!